packages/common/src/configuration/service-configuration.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import * as fs from 'fs';
+import { injectable } from 'inversify';
+import { isNil } from 'lodash';
+import * as defaultConfig from './runtime-config/runtime-config.dev.json';
+
+@injectable()
+export class ServiceConfiguration {
+    public static readonly profilePath = './runtime-configuration/runtime-configuration.json';
+    public config: typeof defaultConfig;
+    private readonly fsObj: typeof fs;
+
+    constructor(fsObj: typeof fs = fs) {
+        this.fsObj = fsObj;
+    }
+
+    public async getConfig(): Promise<typeof defaultConfig> {
+        if (isNil(this.config)) {
+            const fetchConfigPromise = new Promise((resolve, reject) => {
+                this.fsObj.exists(ServiceConfiguration.profilePath, (exists: boolean) => {
+                    if (exists === true) {
+                        this.fsObj.readFile(ServiceConfiguration.profilePath, (err, buffer) => {
+                            if (isNil(err)) {
+                                this.config = JSON.parse(buffer.toString('utf-8')) as typeof defaultConfig;
+                                resolve();
+                            } else {
+                                reject(err);
+                            }
+                        });
+                    } else {
+                        console.log('Unable to find runtime configuration file. Using dev configuration');
```

Comment:
>Using dev configuration [](start = 80, length = 23)

We may need to require mandatory env config ie fail if prod config not available. Can also intro the process var with env type (prod, dev, ppe, etc.)

Reply:
checking for dev.json file instead instead of embedding the file in webpack.